packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -2087,7 +2097,23 @@ export default class Meeting extends StatelessWebexPlugin {
     // Client will have a socket manager and handle reconnecting to mercury, when we reconnect to mercury
     // if the meeting has active peer connections, it should try to reconnect.
     this.webex.internal.mercury.on(ONLINE, () => {
-      LoggerProxy.logger.log('Meeting:index#setReconnectListener --> Web socket online');
+      LoggerProxy.logger.error('Meeting:index#setReconnectListener --> Web socket online');
+
+      // Only send restore event when it was disconnected before and for connected later
+      if (!this.webSocketConnection) {
+        Metrics.postEvent({
+          event: eventType.MERCURY_CONNECTION_RESTORED,
+          meeting: this
+        });
+      }
```

Comment:
We'll want to add a line after this if block to set `this.webSocketConnectionEstablished = true` just in case this is the first time the websocket connection established.